_includes/footer.html

```diff
@@ -14,7 +14,7 @@
         </span>
         <a href="https://twitter.com/ProbotTheRobot" class="twitter-follow-button" data-size="large" data-show-screen-name="false" data-show-count="false">Follow @ProbotTheRobot</a><script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
       </p>
-      <p>{% octicon code %} with {% octicon heart %} by <a class="text-gray-light" href="https://twitter.com/bkeepers">@bkeepers</a></p>
+      <p>{% octicon code %} with {% octicon heart %} by <a class="text-gray-light" href="https://github.com/orgs/probot/people">the Probot team</a></p>
       <p style="opacity: 0.5">
```

Comment:
Good call @gr2m - I changed it to read `the Probot Community`, and link to `https://github.com/probot` (since linking to the website from the website would feel weird).